### PR TITLE
CQA-227: Edited advanced search feature files

### PR DESCRIPTION
**Pull request by cesarvh**
https://issues.collectionspace.org/browse/CQA-227

### CODE REVIEW

**Comment by cesarvh** [comment]
Made various changes affecting different JIRA issues:
- CQA-178: Added step definition to input today's date in a field
- CQA-71& CQA-73-76: These are now working. Please note that the only way to get them to work properly is by zooming all the way out when the browser opens, otherwise it can't find buttons properly
- CQA-221: Made a few fixes to the values of the mappings
- CQA-227: Made changes to the advanced search files to comply with what is specified in the JIRA
  *CQA-228: All the working tests' (71, 73-76) autocomplete-field-related actions are now consolidated

File: src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java [hunk 1]
```diff
@@ -38,6 +44,7 @@
     private Pages pages = new Pages();
     private Selenium selenium;
     private ElementMappings mappings = new ElementMappings();
+    private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
```
Comment (aronr) [inline src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java]: Perhaps 'isoDateFormat' for this variable name, as a disambiguator?

File: src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature [hunk 2]
```diff
@@ -10,10 +10,10 @@ Feature: #Enter feature name here
     Then a drop down list should appear in the "Person" row
     Then the list in the "Person" row should contain "Local Persons, ULAN Persons"
       And clicks on the Create button
-    Then the vocabulary name in the titlebar should contain "Local Person"
+    Then the titlebar should contain "Local Person"
```
Comment (aronr) [inline src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature]: This change and the one on line 18, just below, overwrites recent changes in https://github.com/collectionspace/cspace-cucumber/commit/04f5ab6a9cad62904b27be9cd9e8384a626727d2. Could you verify that this is intended? Thanks!

File: src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java [hunk 3]
```diff
@@ -46,7 +53,27 @@
...
+
+    }
+
+
+
+    @And("^enters today's date in the \"([^\"]*)\" \"([^\"]*)\" field$")
+    public void enters_todays_date_in_field(String recordType, String fieldName) throws Throwable {
+        String date;
+        Date dateNow = new Date();
+        date = dateFormat.format(dateNow);
```
Comment (aronr) [inline src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java]: (See above comment about variable name for date format here ...)

File: src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java (hunk 1 above)
Comment (cesarvh) [inline src/test/java/org/collectionspace/qa/cucumber/stepDefinitions/StepDefs.java]: That sounds much better. Will change in next commit.

File: src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature (hunk 2 above)
Comment (cesarvh) [inline src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature]: I seem to have inadvertently altered this file. Is there a way to reject this change, or should I check it back out from collectionspace/master and re-commit?

Comment (aronr) [inline src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature]: I recall that if you were operating on a feature/topic branch (always a good idea :), simply changing that branch and pushing again would also update the PR. By analogy, it's conceivable that simply making one more local commit, to revert the two changed lines in that feature file, and pushing to cesarvh:master might do the trick? (Am thinking of what might be simplest here, so it isn't necessary for either of us to figure out how to remove just one, inadvertently committed file from an older commit :) )

**Pull request merged**